src/SchemaForm.vue

```diff
@@ -52,7 +51,8 @@ export default {
       default: false
     }
   },
-  setup (props, { emit, attrs }) {
+  emits: ['submit', 'update:modelValue'],
```

Comment:
I don't know why it changed the order. Lemme know if you want it back.

Reply:
Alles gut